Sources/Info.plist

```diff
@@ -2,6 +2,13 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>UIAppFonts</key>
+	<array>
+		<string>FINNTypeWebStrippet-Bold.ttf</string>
+		<string>FINNTypeWebStrippet-Light.ttf</string>
+		<string>FINNTypeWebStrippet-Medium.ttf</string>
+		<string>FINNTypeWebStrippet-Regular.ttf</string>
+	</array>
```

Comment:
Do we still need to do this?

Reply:
I doubt, but why not be explicit?
Also more understandable if you’re reading the info file :)

Reply:
If they are there you expect them to do something, but they are actually not needed. Not sure how that is explicit?

Reply:
good point.